ee/tabby-webserver/src/service/auth.rs

```diff
@@ -403,15 +400,20 @@ impl AuthenticationService for DbConn {
         &self,
         provider: OAuthProvider,
         client_id: String,
-        client_secret: Option<String>,
-        active: bool,
+        client_secret: String,
     ) -> Result<()> {
         match provider {
             OAuthProvider::Github => Ok(self
-                .update_github_oauth_credential(&client_id, client_secret.as_deref(), active)
```

Comment:
overall looks good to me. but you may need to revert this change if we still need to pass `Option<&str>`.

Reply:
for this specific case (github oauth), i think we don't really have a case that need to update client_id only, without updating the client_secret. thus I think it's fine to make client_secret a required field in updating, after removing active.

Reply:
sounds good, no more comments from my side.